app/src/main/res/values/strings.xml

```diff
@@ -1010,7 +1010,7 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_shop_gone_vacant_answer">It’s vacant</string>
     <string name="quest_shop_gone_replaced_answer">A(n)…</string>
     <string name="quest_shop_gone_replaced_answer_hint">Type kind of place here</string>
-    <string name="quest_shop_gone_replaced_answer_error">"Don’t know what this is. Consider leaving a note."</string>
+    <string name="quest_shop_gone_replaced_answer_error2">"Not recognized. Try entering the type of place, for example 'bakery'. Otherwise, consider leaving a note."</string>
```

Comment:
To stay consistent with quotation marks in existing strings:

```suggestion
    <string name="quest_shop_gone_replaced_answer_error2">"Not recognized. Try entering the type of place, for example “bakery”. Otherwise, consider leaving a note."</string>
```